src/app/app.module.ts

```diff
@@ -6,21 +6,34 @@ import { ToastrModule } from 'ngx-toastr';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app.routing';
+
 import { RouterModule } from '@angular/router';
-import { AuthLayoutComponent } from './layouts/auth-layout/auth-layout.component';
-import { CommonLayoutComponent } from './layouts/common-layout/common-layout.component';
-import { ComponentsModule } from './components/components.module';
+import { FeaturesComponent } from './modules/features/features.component';
+import { SharedModule } from './modules/shared/shared.module';
 import { AngularFontAwesomeModule } from 'angular-font-awesome';
-import { JwtInterceptor } from './helpers/jwt.interceptor';
-import { ErrorInterceptor } from './helpers/error.interceptor';
+import { JwtInterceptor } from './interceptors/jwt/jwt.interceptor';
+import { ErrorInterceptor } from './interceptors/error/error.interceptor';
 import { LocalStorageService } from './services/local-storage.service';
-import { PasswordResetComponent } from './pages/password-reset/password-reset.component';
-import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
+import {
+  EnterResetPasswordComponent
+} from './modules/authentication/components/enter-reset-password/enter-reset-password.component';
 import { AuthGuard } from './guards/auth.guard';
 import { NoAuthGuard } from './guards/no-auth.guard';
 import { TermsService } from './services/terms/terms.service';
-import { InternationalPaymentStatusComponent } from 'src/app/pages/payment/international-payment-status/international-payment-status.component';
+import {
+  InternationalPaymentStatusComponent
+} from 'src/app/modules/features/components/payment/international-payment-status/international-payment-status.component';
 import { BrowserModule } from '@angular/platform-browser';
+import { AuthenticationComponent } from './modules/authentication/authentication.component';
+import { TermsPageComponent } from './components/terms/terms.component';
+import { PropertiesComponent } from './components/properties/properties.component';
+import { NoPropertiesComponent } from './components/properties/no-properties/no-properties.component';
+import { FeaturesModule } from './modules/features/features.module';
+import { AuthenticationModule } from './modules/authentication/authentication.module';
+import { PropertyDetailsComponent } from './components/property-details/property-details.component';
```

Comment:
Great job @kdaglas, however, it looks like you you have forgotten to change all imports to absolutes.

Reply:
am working on that they are so many sometimes i lose track, but am working on it